nativeLibs/mupdfModule/jni/CMakeLists.txt

```diff
@@ -254,7 +319,7 @@ ${MUPDF_PATH}/thirdparty/zlib
 #-Dhb_realloc_impl=fz_hb_realloc \
 #-Dhb_free_impl=fz_hb_free \
 
-target_compile_definitions(mupdf_thirdparty PRIVATE -DFT2_BUILD_LIBRARY -DDARWIN_NO_CARBON -DFT_CONFIG_MODULES_H="slimftmodules.h" -DFT_CONFIG_OPTIONS_H="slimftoptions.h" -DHAVE_STDINT_H -DOPJ_STATIC -DOPJ_HAVE_INTTYPES_H -DOPJ_HAVE_STDINT_H -DHAVE_FALLBACK=1 -DHAVE_OT -DHAVE_UCDN -DHB_NO_MT -Dhb_malloc_impl=fz_hb_malloc -Dhb_calloc_impl=fz_hb_calloc -Dhb_realloc_impl=fz_hb_realloc -Dhb_free_impl=fz_hb_free)
+target_compile_definitions(mupdf_thirdparty PRIVATE -DFT2_BUILD_LIBRARY -DDARWIN_NO_CARBON -DFT_CONFIG_MODULES_H="slimftmodules.h" -DFT_CONFIG_OPTIONS_H="slimftoptions.h" -DHAVE_STDINT_H -DOPJ_STATIC -DOPJ_HAVE_INTTYPES_H -DOPJ_HAVE_STDINT_H -DHAVE_FALLBACK=1 -DHAVE_OT -DHAVE_UCDN -DHAVE_FREETYPE -DHB_NO_MT -Dhb_malloc_impl=fz_hb_malloc -Dhb_calloc_impl=fz_hb_calloc -Dhb_realloc_impl=fz_hb_realloc -Dhb_free_impl=fz_hb_free)
```

Comment:
Why did you add -DHAVE_FREETYPE flag?

Reply:
If we will remove this flag, we will get an error about missing function. That is happening because in new version of harfbuzz dependency we now have the ifdef condition: https://github.com/ArtifexSoftware/thirdparty-harfbuzz/blob/07c4f02f8c39f4227887249afbd193859da34a68/src/hb-ft.cc#L32C1-L32C21, so if we will remove this flag, the linking step will fail.